src/app.ts

```diff
@@ -211,3 +219,55 @@ io.use(
 );
 
 socketServer(io);
+
```

Comment:
Somewhere the create proxy middleware guy for your local dev environment has disappeared!

Reply:
No longer needed. Just need to ensure local bucket is public access